### ISSUE TITLE
Inline admonition's title is covered when used in conjunction with a titled code block.

### ISSUE THREAD

**Issue by Ultra-Instinct-05** [squidfunk/mkdocs-material#3836]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

When an code block with a title is used along with an inline admonition, then the title of the codeblock covers the title of the inline admonition.

Please refer to this discussion for screenshots 
https://github.com/squidfunk/mkdocs-material/discussions/3835

### Expected behaviour

When a inline admonition is used in conjunction with a titled code block, then both the admonition title and the code block title should be visible.

### Actual behaviour

When a inline admonition is used in conjunction with a titled code block, then the admonition title is covered by the code block.

### Steps to reproduce

1. Download the attached [zip](https://github.com/squidfunk/mkdocs-material/files/8549520/mkdocs-bug.zip) file that contains an extremely minimal setup. 
2. Create a virtual environment using `virtualenv` and install the dependencies `pip install -r requirements.txt`
3. Activate the environment and run `mkdocs serve` and navigate to `http://localhost:8000`
4. The opened page should have a small demo of the bug.

### Package versions

- Python: `3.9.0`
- MkDocs: `1.3.0`
- Material: `8.2.9`

### Configuration

```yaml
site_name: Test Site
theme:
  name: material
docs_dir: src/
nav:
  - Bug Demo: test.md
markdown_extensions:
  - admonition
  - pymdownx.highlight
  - pymdownx.superfences
```

### System information

- Operating system: Windows 11
- Browser: Firefox Nightly 101.0a1

**Comment by squidfunk** [comment]
Thanks for reporting! However, I think I already fixed it in https://github.com/squidfunk/mkdocs-material/commit/41517023cdf1bca264b68aed613495d9937974cf, as noted in https://github.com/squidfunk/mkdocs-material/discussions/3835#discussioncomment-2623408

**Comment by squidfunk** [comment]
Released as part of 8.2.10.
